unit_testing/cli/test_cli_sct_register_multimodal.py

```diff
@@ -17,7 +18,7 @@ def test_sct_register_multimodal_backwards_compat(script_runner):
     assert ret.stderr == ''
 
 
-def test_sct_register_multimodal_mask_no_checks(tmp_path):
+def test_sct_register_multimodal_mask_files_exist(tmp_path):
```

Comment:
That would definitely be preferable. I had only done the minimum needed to test this specific bug, but you're right, now would probably be a good time to test both.

Reply:
Hmmm... right now, `sct_testing_data` doesn't have the inverse warping field. It has `warp_template2anat.nii.gz`, but no `warp_anat2template.nii.gz`, so this test can't be easily written.

This file _does_ exist in `sct_example_data` so it would be good to unify them somehow. But, that might need to be saved for the future.